jsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "compilerOptions": {
+    "baseUrl": "src"
+  },
+  "include": ["src/**/*"]
```

Comment:
I think what we should do is add directory specific ones like one for ... 
components 
features 
etc